src/mount.coffee

```diff
@@ -315,7 +309,7 @@ class Mounter
     debug "stopping #{@instances.length} instances"
     while @instances.length
       instance = @instances.shift()
-      instance.shutdown()
+      instance.shutdown ->
```

Comment:
async.map is also available, so just use that? 

In this code base the use of async is so basic that I think we should migrate to Promises in one (separate) step.